docs/source/examples.rst

```diff
@@ -0,0 +1,70 @@
+Request Examples
+================
+
+In order to make a request the client simply has to import the ModelCreator class.
+When the ModelCreator is instantiated it takes two arguments.  The first is a 'model' argument
+which should be the name of the metric that you are looking for, and be in a CamelCase style.
+This is used by the library to dynamically import the needed model class.  The second argument is
+a dictionary which must include specific key/value pairs for the library to retrieve the required information.
+
+Here are examples of the dictionaries that can be passed to the ModelCreator as an argument
+to get back information.
+
+Time-Series Model
+-----------------
+This type of request is used for the following models: Battery, Cpu, CPU Core Frequency,
+Energy, FPS, FPS Stability, GPU (Imagination), GPU, Janks, Memory, Network, and Power.
+
+.. code-block:: python
+   :linenos:
+
+   time_series_request = {
+       'session_id': 66d926f47ff5a7a5d853d1058c6305614e1ae6a5,
+       'metric': '/cpu',
+       'detail': '',
+       'params': '',
+       'data': ''
+   }
+
+   creator = ModelCreator(time_series_request)
```

Comment:
Should this read `creator = ModelCreator('Cpu', time_series_request)` ?

Reply:
Yes, yes it should.  Thanks for catching that!  Updated.